adapter/service/outlvlswitch/service.go

```diff
@@ -0,0 +1,203 @@
+package outlvlswitch
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/futurehomeno/fimpgo"
+	"github.com/futurehomeno/fimpgo/fimptype"
+
+	"github.com/futurehomeno/cliffhanger/adapter"
+	"github.com/futurehomeno/cliffhanger/adapter/cache"
+)
+
+// Constants defining important properties specific for the service.
+const (
+	MaxLvl        = "max_lvl"
```

Comment:
Prefix constants with their categories. I think duration property belongs slightly more here than to routing.
<img width="664" alt="image" src="https://user-images.githubusercontent.com/31538120/174730783-7b4b65f2-a005-4c6a-a130-ce8071af2ce1.png">